Integrations/integration-MCAS.yml

```diff
@@ -0,0 +1,265 @@
+                     }
+            return entry
+
+    def mcas_username():
+        res=True
+        description = "unknown"
+        mcas_out = ""
+        mcas_username = demisto.args()['username']
+        url = SERVER + '/api/v1/alerts/'
+        initial_request = SERVER + '/cas/api/v1/autocomplete/entities/?search=' + mcas_username
+        headers = {'Authorization': 'Token ' + TOKEN, 'Content-type': 'application/json'}
+        s = requests.get(url=initial_request, headers=headers)
+        a = yaml.load(json.dumps(s.json()))
+        uid = (a['records'][0]['id'])
+        saas = (a['records'][0]['saas'])
+
+        data = '{"skip":0,"limit":20,"filters":{"resolutionStatus":{"eq":[0]},"entity.entity":{"eq":[{"id":"' + uid + '","saas":' + str(saas) + ',"inst":0}]}},"sortField":"date","sortDirection":"desc","performAsyncTotal":true}'
+        r = requests.post(url=url, headers=headers, data=data, verify=USE_SSL)
+
+        # Use yaml library to reformat unicode
+        j = yaml.load(json.dumps(r.json()))
+        # Obtain json index length
+        x = (len(j['data'])) - 1
+        results = []
+
+        if x == 0:
+            description_exist = j['data'][0].get('description', 'no_data')
+            if description_exist != "no_data":
+                description_raw = (j['data'][0]['description'])
+                description = description_raw.replace('<p>'," ").replace('</p>'," ").replace('<br>'," ")
+                results.append({'Username': mcas_username, 'description': description})
+        else:
+            # Loop through all data fields based on json index length
+            for i in range(0, x):
+                # Check for raw data field
+                description_exist = j['data'][i].get('description', 'no_data')
+                if description_exist != "no_data":
+                    description_raw = (j['data'][i]['description'])
+                    description = description_raw.replace('<p>'," ").replace('</p>'," ").replace('<br>'," ")
+                    results.append({'Username': mcas_username, 'description': description})
+
+        # Print message if no records found
+        if description == "unknown":
+            return "No records found."
+        else:
+            # Export results to context
+            entry = {'Type' : entryTypes['note'],
+                      'Contents' :results,
+                      'ContentsFormat' : formats['json'],
+                      'HumanReadable': tableToMarkdown('MCAS list', results),
+                      'ReadableContentsFormat' : formats['markdown'],
+                      'EntryContext' :{'mcas_username': results}
+                     }
+            return entry
+    try:
+        # The command demisto.command() holds the command sent from the user.
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration test button.
+            demisto.results('ok')
```

Comment:
Please add some HTTP query here, so the test module will actually test something.